main/resource-access.bs

```diff
@@ -75,6 +75,291 @@ Issue: Write Linked Data Platform section.
+
+A given Solid resource MUST NOT be directly associated with more than one
+Descriptive metadata resource.
+
+To create or modify a Shape validation metadata resource, an
+[acl:agent](https://github.com/solid/web-access-control-spec#describing-agents)
+MUST have
+[acl:Write](https://github.com/solid/web-access-control-spec#aclcontrol)
+privileges per the [ACL inheritance
+algorithm](https://github.com/solid/web-access-control-spec#acl-inheritance-algorithm)
+on the resource directly associated with it.
+
+To read a Shape validation metadata resource, an
+[acl:agent](https://github.com/solid/web-access-control-spec#describing-agents)
+MUST have
+[acl:Read](https://github.com/solid/web-access-control-spec#aclcontrol)
+privileges per the [ACL inheritance
+algorithm](https://github.com/solid/web-access-control-spec#acl-inheritance-algorithm)
+on the resource directly associated with it.
+
+A Solid server SHOULD sanity check Shape validation metadata resources upon
+creation or update to restrict invalid changes.
+
+#### Server Managed #### {#rm-managed}
+
+Issue: Identify any metadata the server MUST be required to manage.
+
+A Solid server stores information about a resource that clients can read but not
+change in Server Managed metadata. Examples of Server Managed metadata could
+include resource creation or modification timestamps, identity of the agent that
+created the resource, etc. It MUST be supported as a resource metadata type by
+Solid servers.
+
+A Server Managed metadata resource directly associated with a given resource is
+discovered by the client via the `rel=http://www.w3.org/ns/solid/terms#managed`
+`Link` relation in a `Link` header. Conversely, the resource being described by
+a Server Managed metadata resource is discovered by the client via the
+`rel=describes` `Link` relation in a `Link` header.
+
+A given Solid resource MUST NOT be directly associated with more than one Server
+Managed metadata resource.
+
+To read a Server Managed metadata resource, an
+[acl:agent](https://github.com/solid/web-access-control-spec#describing-agents)
+MUST have [acl:Read](https://github.com/solid/web-access-control-spec#aclread)
+privileges per the [ACL inheritance
+algorithm](https://github.com/solid/web-access-control-spec#acl-inheritance-algorithm)
+on the resource directly associated with it. Modes of access beyond
+[acl:Read](https://github.com/solid/web-access-control-spec#aclread) MUST NOT be
+permitted on a Server Managed metadata resource.
+
+#### Configuration #### {#rm-configuration}
+
+Issue: Identify any configuration metadata parameters that MUST be recognized by
+all implementations.
+
+Configuration metadata is used to store configurable parameters for a given
+resource. For example, whether to maintain
+[Mementos](https://tools.ietf.org/html/rfc7089) for a given resource, or how the
+data within a given resource should be rendered or indexed. It MUST be supported
+as a resource metadata type by Solid servers.
```

Comment:
Is the primary user of the contents of a configuration resource the server? Similar to above re "process", is there a statement missing about the conformance requirement eg. "Server MUSTSHOULDMAY process request payload when configuration resource is created or updated."

Is the information in configuration of interest to clients (put there by other applications)?

Reply:
Configuration type has been removed from submission, now being tracked at #178